modules/plugin/geotiff/src/test/java/org/geotools/gce/geotiff/GeoTiffWriterTest.java

```diff
@@ -803,4 +816,28 @@ private void writeAndRead(GridCoverage2D coverage, Boolean writeNoDataParam, dou
         }
         reader.dispose();
     }
+
+    @Test
+    @SuppressWarnings("PMD.CloseResource")
```

Comment:
Suppressed because the FileImageOutputStream is closed along with the reader, right?

Reply:
Right. it's made on the dispose call